### DISCUSSION TITLE
[Esame 11 Febbraio 2020] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#236] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/0b37e73d-a0d6-47e4-bc6f-baaa00753db9)

**Comment by Elia-Belli** [comment]
**(1)** $B$ viene assolto se è ritenuto innocente da almeno 2 giudici (maggioranza):

$$\mathbb{P}(B\ assolto)=\mathbb{P}(AB\ favorevoli)+\mathbb{P}(AC\ favorevoli)+\mathbb{P}(BC\ favorevoli)+\mathbb{P}(ABC\ favorevoli)$$

> Quando scrivo $AB$ favorevoli, intendo che il terzo è contrario

$$=1/2\cdot [(1/4\cdot 3/4)+(1/4\cdot 1/4)+(3/4\cdot 3/4)+(1/4\cdot 3/4)]=\frac{16}{32}=\frac{1}{2}$$

**(2)** Applicando la probabilità condizionata:

$$\mathbb{P}(B\ assolto|A\ contrario)=\frac{\mathbb{P}(BC\ favorevoli \cap A\ contrario)}{\mathbb{P}(A\ contrario)}=\frac{3}{4}\cdot \frac{1}{2}=\frac{3}{8}$$

**(3)**

$$\mathbb{P}(C\ favorevole| B\ assolto)=\frac{\mathbb{P}(C\ favorevole \cap B\ assolto)}{\mathbb{P}(B\ assolto)}=\frac{1/2\cdot [(1/4\cdot 1/4)+(3/4\cdot 3/4)+(1/4\cdot 3/4)]}{1/2}=\frac{13}{16}$$

**(4)** Secondo le nuove direttive $C$ è favorevole se copia $A$ e $A$ è favorevole oppure se copia $B$ e $B$ è favorevole, ovvero:

$$\mathbb{P}(C\ favorevole)=\frac{1}{4}p+(1-p)\frac{3}{4}=\frac{3}{4}-\frac{1}{2}p$$

> In pratica se $B$ paga na cifra de sordi a $C$ , $p\to 1$ e lui copierà sempre $B$ ovvero il giudice più favorevole

Ora procediamo analogamente al punto **(1)** considerando i vari casi favorevoli:

$$\mathbb{P}(B\ assolto)=(\frac{3}{4}-\frac{1}{2}p)\cdot (1/16+9/16+3/16)+(\frac{1}{4}+\frac{1}{2}p)\cdot (3/16)=\frac{21}{32}-\frac{5}{16}p$$

> Ho raggruppato i casi in cui $C$ è favorevole e non per essere più sintetico
Naturalmente $(\frac{1}{4}+\frac{1}{2}p)$ è la probabilità di $C$ sfavorevole $=1-(\frac{3}{4}-\frac{1}{2}p)$

Possiamo verificare il conto ponendo $p=1/2$ (in pratica sceglie equamente di copiare l'uno o l'altro e mediamente essendo uno favorevole e l'altro contrario il giudizio si rivela equo) e vediamo che si torna alla situazione originaria:

$$\frac{21}{32}-\frac{5}{16}\cdot \frac{1}{2}=\frac{16}{32}=\frac{1}{2}$$

**Comment by LeoCosta02** [comment]
@Elia-Belli il punto 3 se facciamo come abbiamo fatto al punto 2 non dovrebbe venire esattamente la probabilità che C lo dichiari innocente , ovvero 1/2 ?

**Reply by Elia-Belli** [reply]
no perchè c'è un caso in cui $C$ lo dichiara innocente e gli altri due no, quindi non viene assolto in quel caso e allora non può essere la stessa probabilità di $C$
